docs/source/filters.rst

```diff
@@ -403,13 +403,23 @@ project for the latest release version, to be notified of new releases:
 
 .. code:: yaml
 
-   url: https://github.com/thp/urlwatch/releases
+   url: https://github.com/tulir/gomuks/releases
```

Comment:
@thp Was this change of `url` merged intentionally?

Reply:
That makes sense.

Reply:
Yeah, i had to do a double-take too, but as @julianuu said, there are now two "unit tests" for those filters, so the URL needs to be unique. The unit test doesn't do HTTP requests, and just uses the local test data, so it will continue working as an example (for unit tests) even if the repo goes away.

Reply:
Sorry, it was not on purpose that the urls have the same length ;)